cfg/17.06/definitions.yaml

```diff
@@ -453,6 +453,7 @@ groups:
     description: "Ensure Userland Proxy is Disabled (Scored)"
     audit: ps -ef | grep dockerd
     tests:
+      bin_op: or
```

Comment:
Not sure whether `bin_op: or` works if there is only one option, but I'd say it would be easier to read the test if we omit this too